text/0074-structured-vcd.md

```diff
@@ -204,9 +207,13 @@ $upscope $end
 - Should we continue to include the "flattened" [pure bit-vector] representation of aggregate signals in the VCD?
 - Does this feature need to be gated/opt-in by default?
 
+- The simulator currently depends on [westerndigitalcorporation/pyvcd](https://github.com/westerndigitalcorporation/pyvcd) when writing VCD files.
+  However, `pyvcd` only emits scope types defined by the VCD specification (and does not include the `vhdl_record`/`vhdl_array` scopes).
+  When implementing this RFC, should we continue relying on `pyvcd`, or does this warrant the addition of our own code in the simulator for writing VCD files?
```

Comment:
One reason to use pyvcd (VCD itself is not hard to emit) is its support for `.gtkw` files. I really don't want to have more GTKWave specific code in Amaranth...